### PR TITLE
Dropdown menu UI fixes

### PR DESCRIPTION
**Pull request by archiewood**
### Description
This addresses some of the regressions in https://github.com/evidence-dev/evidence/issues/739
- Click outside the menu to close
- Expand header bar to 100% on narrow viewports
- Tone down shadow
- Remove get help in production

### Checklist
- [x] For UI or styling changes, I have added a screenshot or gif showing before & after
- [x] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)

#### Click outside to dismiss
CleanShot 2023-04-10 at 19 25 38

### CODE REVIEW

**Comment by archiewood** [comment]
Todo:
- [ ] Make all drowdown menu items same height (currently `<a>` and `<button>` elements are slightly different heights )

**Comment by archiewood** [comment]
The click outside to dismiss solution feels a bit gnarly. Would take advice on this if there are other thoughts!

**Comment by ItsMeBrianD** [comment]
Regarding click outside, here's a library action: https://www.svelteui.org/composables/use- -click-outside, and here's one that I've worked on previously https://github.com/SprocketBot/sprocket/blob/staging/clients/web/src/lib/utils/actions/clickOutsideAction.ts

**Comment by mcrascal** [comment]
To do: 

- [ ] Look at constraining to article width (e.g. the kebab should be vertically aligned with the right margin of the article, and potentially constraining breadcrumbs alongside it.

**Comment by archiewood** [comment]
I have a version of the header which constrains the menu and breadcrumbs to article width.

### Before
CleanShot 2023-04-12 at 14 57 45

### After
CleanShot 2023-04-12 at 14 54 25


I'm not totally convinced this is better. Some examples:
## Notion
image

## Streamlit
image

I think the problem here we're trying to solve is making the menu more obvious. Open to suggestions, but some ideas:
- Make the icon bigger
- Make the icon darker grey
- Change the icon for something more visually eye-catching
- Move some menu items out of the menu
    - Perhaps show/ hide queries as separate icon
    - A generic share icon, that currently only has one option, Export PDF.

**Comment by ItsMeBrianD** [comment]
@archiewood this may be a helpful tailwind utility for maintaining this behavior:
https://tailwindcss.com/docs/container

**Comment by mcrascal** [comment]
I'm persuaded for full width. 

Is the preview deploy current? Click outside not working for me in safari.

**Comment by archiewood** [comment]
ah yeah, me neither. shoulda tested cross browser.

Feels like @ItsMeBrianD's clickoutside action it probably a more robust way to go

File: sites/example-project/src/components/ui/PageMenu.svelte [hunk 1]
```diff
@@ -27,14 +32,38 @@
...
+		}
+		update({ enabled: initialEnabled });
+		return {
+			update,
+			destroy() {
+				window.removeEventListener('click', handleOutsideClick);
+			}
+		};
+	}
 </script>
```
Comment (ItsMeBrianD) [inline sites/example-project/src/components/ui/PageMenu.svelte:59]: This should get moved to it's own file; it's something that we are probably going to want to use in other places.

**Review by ItsMeBrianD — CHANGES_REQUESTED** [review]
Looks great, just need to extract clickOutside to it's own file and this should be good to go

**Pull request merged**